app/assets/stylesheets/adoption_applications.css.scss

```diff
@@ -0,0 +1,56 @@
+h1 {
+  font-family: 'Architects Daughter', cursive;
+  padding-top: 30px;
+}
+
+.form_questions {
+  @include span-columns(8);
+  @include shift(2);
+}
+
+.radio {
+  padding: 10px 0;
+}
+
+.note {
+  @include span-columns(8);
+  @include shift(0);
+  color: $dark-blue;
+  padding: 10px 0 20px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px 0 20px` instead of `10px 0 20px 0`